src/pages/layout.js

```diff
@@ -57,45 +56,50 @@ const Layout = () => {
     }
 
     return (
-        <HashRouter>
-            <TwoPanel mainStyle={styles.twoPanelMain}>
-                <SideBar />
-                <MainContent>
-                    <Paper className={layoutStyles.paper__layout}>
-                        <Switch>
-                            <Route path="/" exact>
-                                <D2Shim>
-                                    {openFirstLaunch === true ? (
-                                        <DialogFirstLaunch
-                                            handleSave={handleSave}
-                                            onClose={handleClose}
-                                            disable={isDisabledAuthority}
-                                        />
-                                    ) : (
-                                        <Redirect to={menuSection[0].path} />
-                                    )}
-                                </D2Shim>
-                            </Route>
+        <>
+            <CssVariables colors spacers />
```

Comment:
This should probably be placed in [`src/App.js`](https://github.com/dhis2/android-settings-app/blob/master/src/App.js), the entrypoint for the application, instead of here